File: 110algoritmos/exercicios013/ex013-Csharp/ex013.cs
using System;

class Program {
    static void Main() {
        double salario, aumento;

        Console.Write("Qual é o salário do funcionário? R$");
        salario = double.Parse(Console.ReadLine());

        aumento = salario * 1.15;

        Console.WriteLine("Um funcionario que ganharava R$" + salario + " com 15% de aumento, passa a receber R$" + aumento.ToString(2) + ".\n");

    }
}

File: 110algoritmos/exercicios013/ex013-Rust/ex013.rs
use std::io;

fn main() {
    
    let mut salario:String = String::from("");

    println!("Qual é o salário do funcionario? R$");
    io::stdin()
        .read_line(&mut salario)
        .expect("Error ao ler salario");
    let salario:f32 = salario.trim().parse().expect("Erro ao converter salario");

    let aumento = salario * 1.15;

    println!("Um funcionario que ganhava R${}, com 15% de aumento, passa a receber R${:.2}.", salario, aumento);


}